src/cluster_legacy.c

```diff
@@ -2453,8 +2620,16 @@ void clusterUpdateSlotsConfigWith(clusterNode *sender, uint64_t senderConfigEpoc
          *
          * In order to maintain a consistent state between keys and slots
          * we need to remove all the keys from the slots we lost. */
-        for (j = 0; j < dirty_slots_count; j++)
+        for (int j = 0; j < dirty_slots_count; j++) {
+            serverLog(LL_WARNING,
```

Comment:
Lower it to `LL_DEBUG` ?

Reply:
Will lower it to NOTICE. I think the convention is to reserve `WARNING` for errors that are actionable by the "operator". Since this is a potential data loss situation, I think it warrants `NOTICE`.